web/src/components/InlineTextFilter/InlineTextFilter.test.tsx

```diff
@@ -3,19 +3,20 @@ import { render, screen } from "@testing-library/react";
 import { InlineTextFilter } from "./InlineTextFilter";
 
 describe("InlineTextFilter", () => {
-	it("should render label with given text", () => {
-		render(<InlineTextFilter label="Test label" name="test" />);
-
-		const label = screen.getByText("Test label");
+	it("should render Input with given heading", () => {
+		render(<InlineTextFilter name="test" heading="Test heading" />);
+		const label = screen.getByRole("button", {
+			name: "Test heading",
+		});
 
 		expect(label).toBeInTheDocument();
-		expect(label).toHaveProperty("tagName", "LABEL");
+		expect(label).toHaveProperty("tagName", "BUTTON");
 	});
 
-	it("should render label matching given input name", () => {
-		render(<InlineTextFilter label="Test label" name="test" />);
+	it("should render Input matching given input name", () => {
```

Comment:
Maybe change test name to 'should render input with id matching the given component name prop'.